test/hmc.jl

```diff
@@ -2,18 +2,25 @@ using Test, AdvancedHMC
 using Statistics: mean, var, cov
 include("common.jl")
 
-@testset "HMC and NUTS" begin
-    θ_init = randn(D)
-    ϵ = 0.02
-    n_steps = 20
-    n_samples = 100_000
+θ_init = randn(D)
+ϵ = 0.02
+n_steps = 20
+n_samples = 100_000
+n_adapts = 1_000
 
+@testset "HMC and NUTS" begin
```

Comment:
Looping all possible combinations supported now. I manually check `@code_warntype` for all `sample()` functions and they are all stable. Would be nice if you can confirm this from your side (simply by replaces `samples = ...` to `@code_warntype ...`) and run it in REPL. I don't why I can't run it using `julia test/hmc.jl`, which says `code_warntype` not defined.